sdks/java/core/src/test/java/org/apache/beam/sdk/schemas/transforms/JoinTest.java

```diff
@@ -176,6 +182,68 @@ public void testInnerJoinDifferentKeys() {
     pipeline.run();
   }
 
+  @Test
+  @Category(NeedsRunner.class)
+  public void testInnerJoinDifferentKeysNullable2() {
+    List<Row> pc1Rows =
+        Lists.newArrayList(
```

Comment:
Nit: prefer `ImmutableList.of(...)`.

Reply:
This is what I get for copying the previous test and not looking. Updated all of them in this file (which was copied from CoGroupTest).

Reply:
> which was copied from CoGroupTest

Ah, that explains the mysterious `CG_` prefix. We should consider deleting that as well (though it doesn't have to happen in this PR).